src/platform/user/profile/vet360/containers/InitializeVet360ID.jsx

```diff
@@ -64,12 +62,7 @@ class InitializeVet360ID extends React.Component {
   render() {
     switch (this.props.status) {
       case VET360_INITIALIZATION_STATUS.INITIALIZED:
-        return (
-          <div>
-            {!environment.isProduction() && <AddressValidationModal />}
```

Comment:
This was the main thing I wanted to accomplish. Remove this `AddessValidationModal` component from the `InitializeVet360ID` component.